pymc4/utils.py

```diff
@@ -58,6 +58,8 @@ def merge_dicts(*dicts: dict, **kwargs: dict):
 def biwrap(wrapper):
     @functools.wraps(wrapper)
     def enhanced(*args, **kwargs):
+
+        # What does this do? Seems to check if args are passed to decorator?
```

Comment:
@ferrine 
For this biwrap utils, I get that biwrap  decorates `pm.coroutine.model` but what does enhance do specifically?